sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -142,8 +155,11 @@ def get_calibration_files(self):
             calibration_exists = self._check_arm_calibration_exists(arm_serial)
             if not calibration_exists:
                 calibration_exists = self._generate_new_arm_calibration(arm_ip, arm_serial)
-            if calibration_exists:
-                calibration_file_location = os.path.join(self._arm_calibrations_folder, arm_serial + ".yaml")
+            if calibration_exists: 
+                if "yaml" not in arm_serial:
```

Comment:
get serial from yaml should just return a number, not a name of a file, so arm_serial by default should NOT have a yaml suffix at this point. So this line seems redundant to me, unless I'm misunderstanding something.

Reply:
I completely agree on the logic as you are saying. At some point without this edit, I had a .yaml.yaml file and resulted in exception for file loading. Not sure why it happened